Microsoft.Identity.Web/WebApiServiceCollectionExtensions.cs

```diff
@@ -41,13 +41,9 @@ public static class WebApiServiceCollectionExtensions
             string configSectionName = "AzureAD",
             bool subscribeToJwtBearerMiddlewareDiagnosticsEvents = false)
         {
-            services.Configure<AzureADOptions>(options => configuration.Bind(configSectionName, options));
```

Comment:
Why did you remove this one, @TiagoBrenck  ?

Reply:
If you add this line back, you will notice that the `JwtBearerOptions` doesn't get set and the sample doesn't work. I compared with the version before merge and I noticed that we didn't have this line, so after removing it and seeing that the sample worked, I assumed that this line was the problem.

Reply:
That makes sense. There is already the .AddAzureADBearer.